### DISCUSSION TITLE
紧急委托！忘了怎么换头像和昵称了啊啊啊啊啊现在它显示的是未登录

### DISCUSSION THREAD

**Discussion by cuijiaqi0406** [share121/inter-knot#5076] [委托]
![Screenshot_20240917_180100](https://github.com/user-attachments/assets/91d53e33-2b2d-4a3e-aa8a-65c2014c4c45)

**Comment by KELERSAMA** [comment]
关于换头像 可以看帖子 #4990 来自小三月的教程
关于登录 "我的"→"用户接口"

**Comment by cuijiaqi0406** [comment]
换完了，但是app端还是显示自己未登录
![Uploading Screenshot_20240917_183505.jpg…]()

**Reply by KELERSAMA** [reply]
去APP"我的"界面  把公共接口改成用户接口 重新登录一遍
